### PR TITLE
feat/Search feature

### LINKED ISSUE #69: Search Feature
Search Feature

Frontend - @sumeetrohra 
Backend - @sumeetrohra

### PR DESCRIPTION
**Pull request by sumeetrohra**
close #69

## Types of changes

- [x] Add tags to auto close the corresponding issue.
- [x] I followed branch and commit naming convention.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: client/src/components/Header/index.jsx [hunk 1]
```diff
@@ -53,7 +54,7 @@ const Header = () => {
         </MenuItem>
         <HeaderMenuRight>
           <MenuItem>
-            <SearchBox />
+            <SearchBox disabledSearchBox={Boolean(disabledSearchBox)} />
```
Comment (Navneet-Sahota) [inline client/src/components/Header/index.jsx]: `disabledSearchBox`  is already of type boolean. You don't need to coerce it into Boolean again

File: client/src/components/SearchBar/SearchBar.module.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+/*
```
[the remark this answers is not in the document]
Comment (Navneet-Sahota) [reply client/src/components/SearchBar/SearchBar.module.css]: Remove comments

**Comment by Navneet-Sahota** [comment]
Add a gif as well

File: client/src/components/Header/index.jsx (hunk 1 above)
Comment (sumeetrohra) [reply client/src/components/Header/index.jsx]: done

**Pull request merged**